### DISCUSSION TITLE
Install error on Ubuntu 22.04.1

### DISCUSSION THREAD

**Discussion by jstmn** [krishauser/Klampt#156] [Ideas]
Hi,

FYI: The command `make unpack-deps; make deps` was failing for me when I was installing Klampt by source.  I was getting the following error:
```
In file included from glui_add_controls.cpp:37:
./include/GL/glui.h:60:12: fatal error: GL/glut.h: No such file or directory
   60 |   #include <GL/glut.h>
      |            ^~~~~~~~~~~
compilation terminated.
```

Per https://stackoverflow.com/a/65194707/5191069 I installed the following: `sudo apt-get install libglfw3-dev libgl1-mesa-dev libglu1-mesa-dev`

I figured this should be updated in the docs so that others don't need to run in this, or a some other solution should be implemented

Ubuntu version:
```
Description:	Ubuntu 22.04.1 LTS
Release:	22.04
Codename:	jammy
```

Thanks

**Comment by jstmn** [comment]
Moving this to an issue. It looks like github doesn't let you delete discussions for some reason (?)

**Reply by krishauser** [reply]
GLUI is not necessary for building Klampt unless you are building the C++ apps and Qt is not available. The GLUI apps are greatly reduced in functionality, so it is safe to ignore these errors. It may be better to just take GLUI out of the default build path.
